encoding/thrift/register.go

```diff
@@ -39,45 +40,38 @@ func Register(r transport.RouteTable, rs []transport.Procedure) {
 	r.Register(rs)
 }
 
-// UnaryHandler represents a Thrift request handler. It speaks in raw Thrift payloads.
-//
-// Users should use the server package generated by the code generator rather
-// than using this directly.
-type UnaryHandler interface {
-	Handle(ctx context.Context, body wire.Value) (Response, error)
-}
+// UnaryHandler is a convenience type alias for functions that act as Handlers.
+type UnaryHandler func(context.Context, wire.Value) (Response, error)
```

Comment:
Please update the doc. This is not a convenience type alias anymore. It is what UnaryHandler was before.